appveyor.yml

```diff
@@ -20,8 +25,9 @@ shallow_clone: true
 clone_depth: 1
 test_script:
   - node --version
-  - npm --version
+  - yarn --version || npm --version
```

Comment:
As far as I'm aware `||` only runs the second command if the first fails; it isn't immediately obvious (to me anyway) whether `yarn` would not even be available in the case that `%install_command%` is `npm`, so this looks at a glance as though it will always print the yarn version and not the npm version. Presumably we'd want to always print both?